test/extension-tests/multi-root-UI/runTest.ts

```diff
@@ -14,9 +15,11 @@ async function main() {
 
     const testWorkspace = path.resolve(extensionDevelopmentPath, 'test/extension-tests/multi-root-UI/project-workspace.code-workspace');
 
-    const launchArgs = [ "--disable-extensions", testWorkspace ];
-
-    const extensionTestsEnv: { [key: string]: string | undefined } = { "CMT_TESTING": "1" };
+    const launchArgs = ["--disable-extensions", testWorkspace];
+    const extensionTestsEnv: { [key: string]: string | undefined } = {
+      "CMT_TESTING": "1",
+      "HasVs": os.platform().toLowerCase().includes("win") ? "true" : "false"
```

Comment:
I don't see `HasVs` used anywhere in the product.  You can probably remove this variable from the test configs.